### DISCUSSION TITLE
Ability to see the previous beatmap from Osu! BOTD

### DISCUSSION THREAD

**Discussion by MoutonBinoclard** [ppy/osu#29258] [Ideas] [answered]
It would be great to see which map was played yesterday, or since the begining of BOTD
Like when you're in the BOTD tab, being able to click a button and seeing all the old map, don't you think ?

**Comment by peppy** [comment]
We're adding this to osu!web this week!

**Reply by MoutonBinoclard** [reply]
Is it up now or not yet ?

**Reply by SupDos** [reply, marked as answer]
https://osu.ppy.sh/rankings/daily-challenge/
